source/codegen/metadata/niscope/functions.py

```diff
@@ -1458,7 +1384,41 @@
         ],
         'returns': 'ViStatus'
     },
+    'ErrorMessage': {
```

Comment:
Wasn't here before. I think we'll just 'codegen_method': 'no' for this function in hapigen.